### PR TITLE
💅 Add a config for the Chronographer GitHub App

### PR DESCRIPTION
**Pull request by webknjaz**
This app allows requiring changelog fragments to be included with each pull request.

### CODE REVIEW

File: .github/chronographer.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  external-docs-url: >-
+    https://docs.pytest.org/en/latest/contributing.html#preparing-pull-requests
+  inline-markdown: >-
+    See
+    https://docs.pytest.org/en/latest/contributing.html#preparing-pull-requests
+    for details.
+enforce-name:
+  suffix: .rst
+# labels:
+#   skip-changelog: skip news
```
Comment (webknjaz) [inline .github/chronographer.yml:15]: @RonnyPfannschmidt plz create a `bot:chronographer:skip` label. Alternatively, we can uncomment this part and use a nicer name. `skip news` is what pip uses.

**Comment by webknjaz** [comment]
@RonnyPfannschmidt the app that needs to be installed is https://github.com/apps/psf-chronographer/installations/new/permissions?target_id=8897583&target_type=Organization. You can do it right away.

File: .github/chronographer.yml (hunk 1 above)
Comment (RonnyPfannschmidt) [reply .github/chronographer.yml:18]: https://github.com/pytest-dev/pytest/labels/skip%20news was created

Comment (webknjaz) [reply .github/chronographer.yml]: I've uncommented the config section. Marking the thread as resolved.

**Comment by webknjaz** [comment]
@RonnyPfannschmidt applying that label here will not yet work until this is merged to `main`. The app doesn't read configs from PRs.

**Review by RonnyPfannschmidt — APPROVED** [review]

**Comment by webknjaz** [comment]
@nicoddemus FYI from now on, the PRs that don't need changelog fragments must be marked with the `skip news` label for the required check to pass.

**Pull request merged**